### ISSUE TITLE
new coq/ocaml compat

### ISSUE THREAD

**Issue by vzaliva** [Karmaki/coq-dpdgraph#69]
I have coq-8.9.1 and ocaml 4.09.0:

```
 $ opam install coq-dpdgraph                                                         16:43:28
The following dependencies couldn't be met:
  - coq-dpdgraph → coq < 8.7~ → ocaml < 4.06.0
      base of this switch (use `--unlock-base' to force)
  - coq-dpdgraph → ocaml < 4.08.0
      base of this switch (use `--unlock-base' to force)
```

**Comment by ybertot** [comment]
I am looking into it.

**Comment by ybertot** [comment]
The problem should soon be fixed by release v0.6.5, which will be taken into account in the opam coq archive as soon as PR#1096 is merged there (it is currently under review).
corrective actions have also been taken so that the problem does not appear for later versions of coq.

**Comment by ybertot** [comment]
The problem should be solved now.
